website/frontend/templates/docs/build-windows.html

```diff
@@ -125,30 +204,98 @@ <h3 id="nsis">NSIS 2.46</h3>
+      </p>
+        
+      <h4 id="visual-cpp-py3">Python 2 - Visual C++ 2015 Community</h4>
+
       <p>
-        Then you need to copy msvcr90.dll and msvcp90.dll from "C:\Program Files\Microsoft Visual Studio 9.0\VC\redist\x86\Microsoft.VC90.CRT"
-        to the source code directory, so that py2exe can find them. You also need copy discid.dll (libdiscid)
-        to the picard source.
+        For Picard version 2 / Python 3, 
+        download <a rel="nofollow" href="http://landinghub.visualstudio.com/visual-cpp-build-tools">Microsoft Visual C++ 14.0 
+        standalone: Visual C++ Build Tools 2015 (x86, x64, ARM)</a> from http://landinghub.visualstudio.com/visual-cpp-build-tools.
       </p>
+        
+      <p>
+        When you install, select the Windows 8.1 SDK for compile to work.
+      </p>
+        
+      <h4 id="visual-cpp-py2">Python 2 - Visual C++ 2008 Express</h4>
+
+      <p>
+        For Picard version 1 / Python 2, 
+        download <a rel="nofollow" href="http://download.microsoft.com/download/A/5/4/A54BADB6-9C3F-478D-8657-93B3FC9FE62D/vcsetup.exe">Visual C++ 2008 Express</a>. 
+      </p>
+        
+      <p>
+        <b>Note:</b> There is also apparently a cut-down compiler specifically for compiling Python 2.7.x modules available at 
+        <a rel="nofollow" target="_blank" href="https://www.microsoft.com/en-gb/download/details.aspx?id=44266">https://www.microsoft.com/en-gb/download/details.aspx?id=44266</a>,
+        though the author has not tried this.
+      </p>
+
+      <h3 id="visual_cpp">Visual C++</h3>
+      
+      <p>
+        Before you can build the Python executable and installer, you need to build the astrcmp C extension and 
+        language internationalisation. 
+        Start the VS console using "Visual Studio 2008 Command Prompt" or "Visual C++ 2015 MSBuild Command Prompt", 
+        go to the Picard source directory and run these commands:
+      </p>
+
+      <pre>python setup.py build_ext -i
+python setup.py build_locales -i</pre>
+
+      <p>
+        Then you need to copy the following files from Visual Studio to 
+        to the source code directory, so that py2exe can find them:
+      </p>
+        
+      <ul>
+        <li>
+          Picard v2 / Python 3 - msvcp140.dll from 
+          <i>C:\Program&nbsp;Files&nbsp;(x86)\Microsoft&nbsp;Visual&nbsp;Studio&nbsp;14.0\VC\redist\x86\Microsoft.VC140.CRT</i>
+        </li>
+        <li>
+          Picard v1 / Python 2 - msvcr90.dll and msvcp90.dll from 
+          <i>C:\Program&nbsp;Files&nbsp;(x86)\Microsoft&nbsp;Visual&nbsp;Studio&nbsp;9.0\VC\redist\x86\Microsoft.VC90.CRT</i>
+        </li>
+      </ul>
+        
+      <p>
+        You also need copy discid.dll (libdiscid) to the picard source.
+      </p>
+        
       <p>
         Then you can build the installer using this command:
       </p>
-      <pre>
-        python setup.py bdist_nsis</pre>
+      <pre>python setup.py bdist_nsis</pre>
```

Comment:
Just checking, since there seems to be a difference in how pip is called: is `python` always either Python 3 or Python 2, depending on which was installed?